From f93093da67f995af4744c7f6142c03b10926acb5 Mon Sep 17 00:00:00 2001
From: Patrick Williams <patrick@stwcx.xyz>
Date: Fri, 26 May 2023 15:14:02 -0500
Subject: [PATCH] Fix GCC 13 issue

GCC identifies an uninitialized local variable that is later used.
Initialize the variable to a sane status.

Signed-off-by: Patrick Williams <patrick@stwcx.xyz>
Change-Id: I77a80a7c7e76c8964dfd4fb2d179aab3d7f03b03
---
 peci_cmds.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/peci_cmds.c b/peci_cmds.c
index 24c3d8d..064e013 100644
--- a/peci_cmds.c
+++ b/peci_cmds.c
@@ -99,7 +99,7 @@ int main(int argc, char* argv[])
     int c;
     int i = 0;
     char* cmd = NULL;
-    EPECIStatus ret;
+    EPECIStatus ret = PECI_CC_SUCCESS;
     uint8_t address = 0x30; // use default address of 48d
     uint8_t u8Size = 4;     // default to a DWORD
     uint32_t u32PciReadVal = 0;
@@ -928,7 +928,7 @@ int main(int argc, char* argv[])
         }
         if (!verbose)
         {
-            if (0 != ret)
+            if (PECI_CC_SUCCESS != ret)
             {
                 printf("ERROR %d: command failed\n", ret);
             }
-- 
2.39.3

